_schemaV2.graphql

```diff
@@ -2287,6 +2287,24 @@ enum ArtistSorts {
   TRENDING_DESC
 }
 
+# An artists rail section in the home view
+type ArtistsRailSection implements GenericHomeViewSection & Node {
```

Comment:
Wondering if these sections also should be prefixed with `HomeView`

Reply:
Fixed in 593d9dabc